pages.es/linux/tcpkill.md

```diff
@@ -0,0 +1,7 @@
+# tcpkill
+
+> Mata la conexiones TCP en curso indicadas.
```

Comment:
```suggestion
> Mata la conexiones TCP en curso especificadas.
```

Reply:
@quique what about the other suggestions?

Reply:
"indicadas" sounds more natural to me.
Does "especificadas" sound better in LatAm?

Reply:
for me, yes. Also, we use `especificado` in the translations.